include/glow/Graph/Graph.h

```diff
@@ -1260,10 +1260,11 @@ class Function final : public Named {
   /// Given \p input tensor of [N,H,W,C], where N is the batch, C is the channel
   /// or depth, H is the height and W is the width, generates an Output tensor
   /// with resized spatial dimensions using nearest neighbor interpolation. The
-  /// Output tensor is of shape [N, floor(H * \p heightScale), floor(W * \p
-  /// widthScale), C]
+  /// Output tensor is of shape [floor(N * \p batchScale),
+  /// floor(H * \p heightScale), floor(W * \p widthScale),
+  /// floor(C * \p channelScale)]
```

Comment:
nit: Thanks for adding to the doxygen, but it's a little confusing since each `xScale` doesn't exist as a parameter. Can you improve this to mention `\p scale` only?

Reply:
Done